lms/templates/courseware/tab_default.html

```diff
@@ -0,0 +1,19 @@
+## mako
+<%namespace name='static' file='/static_content.html'/>
+<%!
+from django.utils.translation import ugettext as _
+from django.core.urlresolvers import reverse
+%>
+<li>
+    <a href="${tab.link_func(course, reverse) | h}" class="${tab_class}">
+    ${_(tab.name) | h}
+    % if tab_is_active:
+        <span class="sr">, current location</span>
```

Comment:
I think it shouldn't. Because as the previous one this behavior also [was before](https://github.com/edx/edx-platform/blob/master/lms/templates/courseware/course_navigation.html#L95) the current pull request. And it seems that developers have made it for a some reason. I don't know why but it seems that it is make sense.